templates/default/rt/rtticketinfobox.html

```diff
@@ -242,7 +242,7 @@
 								</TD>
 							</TR>
 							{/if}
-							{if $ticket.priority}
+							{if !is_null($ticket.priority)}
```

Comment:
tuitaj to samo co wcześniej

Reply:
I tu również da się tak uprościć. Poprawiłem.